joomla/joomla-cms
# [com_fields] Trigger onContentPrepare in news module

Pull Request for Issue #13705.

### Testing instructions
- Create some custom fields for articles
- Edit an article
- Create a news flash module
- Make an override
- Check if the articles do have a $item->fields property available
